src/store/middleware/inputMiddleware.js

```diff
@@ -41,6 +41,16 @@ const inputMiddleware = store => next => action => {
         break;
       }
 
+      case messageReceivedFromRover.type: {
+        const { message } = action.payload;
+        if (message.type === "armIKEnabledReport") {
+          if (store.getState().input.inverseKinematics.lastSentArmIKState !== message.enabled) {
+            alert("Arm IK was unable to be " + (!message.enabled ? "enabled." : "disabled."));
+          }
```

Comment:
Let's modify the `if` to check that `lastSentArmIKState` is nonnull, since `null` will indicate that we haven't requested one, so no point in alerting.